gc/base/standard/Scavenger.hpp

```diff
@@ -260,6 +260,20 @@ class MM_Scavenger : public MM_Collector
 
 	MMINLINE bool copyAndForward(MM_EnvironmentStandard *env, volatile omrobjectptr_t *objectPtrIndirect);
 
+	/**
```

Comment:
Please update the copyright on this file.

Reply:
updated